src/ai/susi/server/api/aaa/GetAAAFiles.java

```diff
@@ -0,0 +1,43 @@
+package ai.susi.server.api.aaa;
```

Comment:
Please add a license header, copy it from the other files and set your own name as copyright holder